app/views/credits/new.html.erb

```diff
@@ -144,7 +144,7 @@
 </div>
   <script>
 
-    var stripe = Stripe('<%= ApplicationConfig["STRIPE_PUBLISHABLE_KEY"] %>');
+    var stripe = Stripe('<%= SiteConfig.stripe_publishable_key %>');
```

Comment:
If an admin hasn't set `SiteConfig.stripe_publishable_key` (since it's not required) , this JavaScript will go boom when try to `createCardElement`, right? Do we want to warn admins about this at all, or are we okay letting [this line](https://github.com/forem/forem/pull/9862/files#diff-919744afb1f282f44b4ebee18b262ef3R206) explode with a `can't call createToken on undefined` error?

Reply:
Yeah, the clean way to do this is to ensure this path never gets hit if this doesn't get shipped. But this is essentially already broken so it's no less broken this way.